docs/Oracle Jobs/jobs.md

```diff
@@ -0,0 +1,272 @@
+
+#### Oracle node
+
+Oracle nodes, on the other hand, are responsible for submitting answers. 
+
+```toml
+type               = "offchainreporting"
+schemaVersion      = 1
+name               = "web oracle spec"
+contractAddress    = "0x613a38AC1659769640aaE063C651F48E0250454C"
+p2pPeerID          = "12D3KooWApUJaQB2saFjyEUfq6BmysnsSnhLnY5CF9tURYVKgoXK"
+p2pBootstrapPeers  = [
+    "/dns4/chain.link/tcp/1234/p2p/16Uiu2HAm58SP7UL8zsnpeuwHfytLocaqgnyaYKP8wu7qRdrixLju",
+]
+isBootstrapPeer    = false
+keyBundleID        = "7f993fb701b3410b1f6e8d4d93a7462754d24609b9b31a4fe64a0cb475a4d934"
+monitoringEndpoint = "chain.link:4321"
+transmitterAddress = "0xF67D0290337bca0847005C7ffD1BC75BA9AAE6e4"
+observationTimeout = "10s"
+blockchainTimeout  = "20s"
+contractConfigTrackerSubscribeInterval = "2m"
+contractConfigTrackerPollInterval = "1m"
+contractConfigConfirmations = 3
+observationSource = """
+    // data source 1
+    ds1          [type=bridge name=voter_turnout];
+    ds1_parse    [type=jsonparse path="one,two"];
+    ds1_multiply [type=multiply times=1.23];
+
+    // data source 2
+    ds2          [type=http method=GET url="https://chain.link/voter_turnout/USA-2020" requestData="{\\"hi\\": \\"hello\\"}"];
+    ds2_parse    [type=jsonparse path="three,four"];
+    ds2_multiply [type=multiply times=4.56];
+
+    ds1 -> ds1_parse -> ds1_multiply -> answer1;
+    ds2 -> ds2_parse -> ds2_multiply -> answer1;
+
+    answer1 [type=median                      index=0];
+    answer2 [type=bridge name=election_winner index=1];
+"""
+```
+
+- `p2pPeerId`: the base58-encoded libp2p public key of this node.
+- `p2pBootstrapPeers`: a list of libp2p dial addresses of the other bootstrap nodes helping oracle nodes find one another on the network.
+- `keyBundleID`: the hash of the OCR key bundle to be used by this node (the Chainlink node keystore manages these key bundles — use the subcommands under `chainlink keys ocr` to interact with them or use the Operator UI).
+- `monitoringEndpoint`: the URL of the telemetry endpoint to send OCR metrics to.
+- `transmitterAddress`: the Ethereum address from which to send aggregated submissions to the OCR contract.
+- `observationTimeout`: the maximum duration to wait before an off-chain request for data is considered to be failed/unfulfillable.
+- `blockchainTimeout`: the maximum duration to wait before an on-chain request for data is considered to be failed/unfulfillable.
+- `contractConfigTrackerSubscribeInterval`: the interval at which to retry subscribing to on-chain config changes if a subscription has not yet successfully been made.
+- `contractConfigTrackerPollInterval`: the interval at which to proactively poll the on-chain config for changes.
+- `contractConfigConfirmations`: the number of blocks to wait after an on-chain config change before considering it worthy of acting upon.
+
+### VRF
+
+```toml
+uuid = "123e4567-e89b-12d3-a456-426655440000"
```

Comment:
this is now called ```externalJobID``` and is a universal parameter across all V2 specs. Probably deserves its own little blurb something like 

The externalJobID is a optionally user-specified way to deterministically provide an ID of a job. Say for example you want to run the same directrequest job on two different chainlink nodes which have different bridge names. Although the actually spec contents differ slightly, you can use the same externalJobID on both, specify that in your on-chain requests and both nodes will pick it up. If you do not provide one, one will be generated by the node.